README.md

```diff
@@ -58,6 +58,15 @@ func main() {
 }
 ```
 
+NonBlocking Mode
+----------------
+	
+```go	
+	c := &serial.Config{Name: "COM45", Baud: 115200, NonBlockingRead: true, ReadTimeout: 1000}
+	// On Windows, ReadTimeout is in milliseconds (1000 = 1sec)
```

Comment:
ReadTimeout should be a time.Duration and the os specific files should translate to their own units.